pkg/ccl/backupccl/backup_job.go

```diff
@@ -810,15 +809,13 @@ func (b *backupResumer) deleteCheckpoint(
 		defer exportStore.Close()
 		// The checkpoint in the base directory should only exist if the cluster
```

Comment:
nit: comment is outdated, can we update it to say that we first attempt to delete from base directory to account for older backups, and then from the progress directory